beacon_node/http_api/src/block_id.rs

```diff
@@ -250,6 +251,37 @@ impl BlockId {
             }
         }
     }
+
+    /// Return the `BlobSidecarList` identified by `self`.
+    pub async fn blob_sidecar_list<T: BeaconChainTypes>(
```

Comment:
This function doesn't need to be async.

Reply:
removed here: 5f958781a0b922739e18eefab52eee4b4ad8d13b